iocore/net/SSLUtils.cc

```diff
@@ -2159,7 +2123,7 @@ ssl_callback_session_ticket(SSL *ssl, unsigned char *keyname, unsigned char *iv,
   ssl_ticket_key_block *keyblock = NULL;
   if (cc == NULL || cc->keyblock == NULL) {
     // Try the default
-    keyblock = global_default_keyblock;
+    keyblock = params->default_global_keyblock;
```

Comment:
Oh I see. I was thinking about the keyblock being attached to the SSL context. You are right, we should hold the scoped config across the function (if we need it). Worth adding a comment to that effect.

It seems unfortunate that we always how a scoped refcount even if we don't need it; we could alleviate that with some restructuring later though.